assets/scripts.js

```diff
@@ -0,0 +1,35 @@
+(function ($) {
```

Comment:
This should go into the scripts folder and be named responsive_video.js
Then you run gulp in the folder and it will add it to the scripts.js folder.
But at the core we would like it added as its own file.